feat(perf-issues): General documentation updates for Performance Issues (#5632)
getsentry/sentry-docs: src/docs/product/data-management-settings/event-grouping/index.mdx

If the stack trace is not available, but exception information is, then the grouping will consider the `type` and `value` of the exception if both pieces of data are present on the event. This grouping is a lot less reliable because of changing error messages.

### Grouping by Span Evidence

If the event is a transaction and it causes a [performance issue](/product/issues/performance-issues/), Sentry fingerprints by the span evidence related to the performance problem found in the event. This includes the spans directly involved in the problem, as well as spans that might have caused the problem or are closely related to it. Note that this algorithm does not support versioning.

> 

Reviewer: This is tricky to just add in here without making any other changes to this page. The first sentence of the page refers explicitly to errors. I think we need to do more work generalizing the content on this page and then probably making note of which parts of the content don't apply to performance issues.

Grouping falls back to messages if the stack trace, `type`, and `value` are not available. When this happens, the grouping algorithm will try to use the message without any parameters. If that is not available, the grouping algorithm will use the full message attribute.

## Transaction Grouping Algorithms

> Sentry groups transactions with performance problems into [performance issues](/product/issues/performance-issues/) using the transaction's `spans`. Sentry fingerprints by the span evidence related to the performance problem found in the event. This includes the spans directly involved in the problem, as well as spans that might have caused the problem or are closely related to it. This algorithm is not customizable or extendable at this time.

Reviewer: ```suggestion
Sentry groups transactions with performance problems into [performance issues](/product/issues/performance-issues/) using the transaction's `spans`. Sentry fingerprints by the span evidence related to the performance problem found in the event. This includes the spans directly involved in the problem, as well as spans that might have caused the problem or are closely related to it. This algorithm isn't currently customizable or extendable.
```
